tutorial/index.jade

```diff
@@ -1,10 +1,155 @@
+
+      1. The user begins the registration process by entering their data, including a phone number, into a signup form
+
+      1. The authentication system sends a one-time password to the user's mobile phone to verify their possession of that phone number
+
+      1. The user enters the one-time password into a form before completing registeration
+
+      1. The user sees a success page and receives an SMS indicating that their account has been created
+
+      ## Building Blocks
+
+      To get this done, you'll be working with the following Twilio-powered APIs:
+
+      **Authy REST API**
+
+      * [Users Resource](https://docs.authy.com/#section-Enabling_two-factor_on_a_user): You will need to create Authy users to send and verify one-time passwords with them.
+      * [SMS Resource](https://docs.authy.com/#section-Text_Messages): We will ask Authy to send one-time passwords to our user via SMS
+      * [Verify Resource](https://docs.authy.com/#section-Verifying_a_token): Used to verify tokens entered by the user in our web form during registration
+
+      **Twilio REST API**
+      * [Messages Resource](/docs/api/rest/sending-sms): We will use Twilio directly to send our user a confirmation message after they create an account
+
+      Let's get started! Click the right arrow to move on to the next step of the tutorial.
+
+  .step(data-title='Registering a User Account', data-file='models/User.js', data-highlight='12-40')
+    :markdown
+      ## The User Model
+
+      Our first order of business is to create a model object for a user of our application. We will borrow a lot of the code from the `User` model in the [2FA tutorial](/docs/howto/walkthrough/two-factor-authentication/node/express) that uses Authy as well. This application uses [MongoDB](http://www.mongodb.com/) for persistence, but in our code we will primarily interface with [Mongoose](http://mongoosejs.com/), a higher-level object modeling tool which is backed by MongoDB.
+
+
+      You'll notice an `authyId` property on the model - [this is required to support integration with the Authy API](https://docs.authy.com/#section-Enabling_two-factor_on_a_user). We won't use this property right away, but we'll need it later for the Authy integration.
+
+      One of the properties on the `User` model, per the user story, is a password (we'll probably want that later to log a user in, even though it's not in scope for this tutorial).
+
+  .step(data-title='The Form Template', data-file='views/users/create.jade', data-language='jade')
+    :markdown
+      ## The Form Template
+
+      When we create a new user, we ask for a name, e-mail address, and a password. 
+      In order to validate their account we also ask them for a mobile number with a country code. 
+      We use Authy to send them a one-time password via SMS to this phone number.
+     
+      Now the user is logged in but not verified. In the next steps we'll learn how to verify the user using authy.
```

Comment:
Usually is a good practice to add references to increase the visibility to the tutorial. That's why we have the _see also_ section in most of the steps. Would you mind adding a reference to [Jade](http://jade-lang.com) please?